docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -47,6 +57,22 @@ If there is an error please [contact support](mailto:pulsesupport@codacy.com).
 -   The set of changes in a deployment is the list of commits in the corresponding pull request.
 -   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
 
+## Automatic incident detection strategies {: id="incident-detection-strategy"}
+
+The following is a detailed description of how the Pulse Bitbucket integration automatically detects incidents:
+
+### Use pull request reverts (based on default branch)
+
+-   Pulse considers an incident a reverted pull request that **targets the default branch** of the repository when the revert operation was executed through the Bitbucket UI originating a branch which name starts with `revert-`.
+-   The incident creation date is the timestamp when the reverted pull request was initially merged.
```

Comment:
This is only true if we know what is the reverted pull request. If the user did not change the branch name for the revert PR, we will know what is the reverted pull request. Maybe we should advise against changing the branch name in the bullet point above?

If we can't know what is the reverted PR, we will consider the incident to have started when the first commit of the incident PR was made.